oak/server/rust/oak_loader/src/main.rs

```diff
@@ -64,6 +65,11 @@ pub struct Opt {
         help = "PEM encoded X.509 TLS root certificate file used to authenticate an external gRPC service."
     )]
     root_tls_certificate: String,
+    #[structopt(
+        long,
+        help = "Path to the downloaded JSON encoded client identity file for OpenID Connect. (Optional)"
```

Comment:
Does it not show up in `--help` already whether a flag is optional? Perhaps it's more useful to know **what** happens when not provided.

Reply:
It does not say whether it is optional, but I updated it with more info.